src/Containers/Lobby/index.tsx

```diff
@@ -1,14 +1,19 @@
-import { useState } from 'react';
+import { useContext } from 'react';
 import { Col, Row } from 'antd';
+import { CrownOutlined } from '@ant-design/icons';
 
 import { Box, CenteredContainer, CenteredDiv, Tags } from 'Components/CommonComponents';
 import Button from 'Components/Button';
-import { PlayerList, StartButton } from './styles';
+import { PlayerEntry, PlayerList, StartButton } from './styles';
+
+import Resources from '../../resourceContext';
```

Comment:
You can set absolute imports in Webstorm settings in Code style for Typescript -> imports -> check second option for paths relative to tsconfig
```suggestion
import Resources from 'resourceContext';

import { PlayerEntry, PlayerList, StartButton } from './styles';
```